### PR TITLE
EPMEDU-1576: Create UI for displaying attached photos

### PR DESCRIPTION
**Pull request by n3gbx**
- Implement Mark Feeding Done bottom sheet
- Display sheet when the feeding is active
- Scroll-left and Scroll-right actions for the list of taken photos
- "Finish" button is inactive until at least one photo hasn't been taken

https://user-images.githubusercontent.com/17089577/223723350-113e1420-8d3c-4b39-837e-dfed36a861e3.mp4

### CODE REVIEW

File: feature/home/src/main/java/com/epmedu/animeal/home/presentation/HomeScreenUI.kt [hunk 1]
```diff
@@ -110,21 +112,32 @@ internal fun HomeScreenUI(
...
             }
         },
         sheetControls = {
             state.currentFeedingPoint?.let { feedingPoint ->
-                FeedingPointActionButton(
-                    alpha = buttonAlpha,
-                    enabled = feedingPoint.feedStatus == FeedStatus.RED,
-                    onClick = { onScreenEvent(WillFeedEvent.ShowWillFeedDialog) }
-                )
+                if (state.feedingRouteState is FeedingRouteState.Active) {
```
Comment (Fileek) [inline feature/home/src/main/java/com/epmedu/animeal/home/presentation/HomeScreenUI.kt]: Please convert to when

File: feature/home/src/main/java/com/epmedu/animeal/home/presentation/viewmodel/HomeState.kt [hunk 2]
```diff
@@ -15,6 +16,7 @@ data class HomeState(
     val currentFeedingPoint: FeedingPointModel? = null,
     val feedingPoints: ImmutableList<FeedingPointModel> = persistentListOf(),
     val feedingRouteState: FeedingRouteState = FeedingRouteState.Disabled,
+    val feedingPhotos: List<Uri> = listOf(),
```
Comment (Fileek) [inline feature/home/src/main/java/com/epmedu/animeal/home/presentation/viewmodel/HomeState.kt]: Could be replaced with `emptyList()` function

**Review by Fileek — CHANGES_REQUESTED** [review]
Please also do following things:
1. Fix expanded bottom sheet when feeding flow is disabled
2. Fix detekt
3. Rebase onto develop branch (we fixed lags)
4. Record demo again

File: feature/home/src/main/java/com/epmedu/animeal/home/presentation/HomeScreenUI.kt (hunk 1 above)
Comment (n3gbx) [reply feature/home/src/main/java/com/epmedu/animeal/home/presentation/HomeScreenUI.kt]: Updated

File: feature/home/src/main/java/com/epmedu/animeal/home/presentation/viewmodel/HomeState.kt (hunk 2 above)
Comment (n3gbx) [reply feature/home/src/main/java/com/epmedu/animeal/home/presentation/viewmodel/HomeState.kt]: Updated

**Comment by n3gbx** [comment]
> Please also do following things:
> 
> 1. Fix expanded bottom sheet when feeding flow is disabled
> 2. Fix detekt
> 3. Rebase onto develop branch (we fixed lags)
> 4. Record demo again

1,2,3 - done. 
4 - I can't record it again due to the absence of available feeding points. If you know how to reset their state please let me know

**Review by Fileek — CHANGES_REQUESTED** [review]
1. Bottom sheet is not showing after starting feeding flow and even after canceling.
2. To reset feeding point status, you need to wait 24 hours or do it manually in DynamoDB.

**Review by Fileek — APPROVED** [review]

**Pull request merged**